lib/bolt/outputter/json.rb

```diff
@@ -21,6 +21,8 @@ def handle_event(event)
         case event[:type]
         when :node_result
           print_result(event[:result])
```

Comment:
I think that maybe subscribing the json outputter prints intermediate node results. (disclaimer this is the last comment I am writing before a long vacation, so I could be way off).